assets/js/booru.ts

```diff
@@ -263,7 +263,4 @@ export function loadBooruData() {
   }
 }
 
-window.booru = {
-  hiddenTag: hiddenTagPath,
-  tagsVersion,
-} as BooruObject;
+window.booru = {} as BooruObject;
```

Comment:
imagesclientside.ts needs to be adjusted to import the new constant hiddenTagPath